data/json/construction.json

```diff
@@ -149,8 +149,20 @@
     "required_skills": [ [ "survival", 1 ] ],
     "time": "60 m",
     "qualities": [ [ { "id": "DIG", "level": 2 } ] ],
-    "components": [ [ [ "material_soil", 50 ] ] ],
-    "pre_terrain": [ [ [ "t_swater_sh", "t_water_sh" ] ] ],
+    "components": [ [ [ "material_soil", 60 ] ] ],
+    "pre_terrain": "t_water_sh",
+    "post_terrain": "t_dirt"
+  },
+  {
+    "type": "construction",
+    "description": "Fill Salt Water With Dirt",
+    "//": "Fills a salt water with dirt.",
```

Comment:
This comment just repeats the description of the construction. Why is it there? Same with the comment of the other construction.

Reply:
I don't know how to put two required terrain options in the same project.

Reply:
I'm not concerned about the two constructions (that is fine, a limitation of the game).

I'm talking about the comments: the `"//": "..."` things. They are comments, so they are ignored by program anyway. In other words: they are unnecessary as (in their current state) they don't add any information. I suggest you remove them.

Reply:
Is it good enough for merging now?